docs/ops/convolution/DeformableConvolution_1.md

```diff
@@ -4,85 +4,180 @@
+  * **Description**: *dilations* denotes the distance in width and height between elements (weights) in the filter. For example, *dilation* equal `1,1` means that all the elements in the filter are neighbors, so it is the same as for the usual convolution. *dilation* equal `2,2` means that all the elements in the filter are matched not to adjacent elements in the input matrix, but to those that are adjacent with distance 1.
+  * **Range of values**: integer value starting from `0`
   * **Type**: int[]
   * **Default value**: None
   * **Required**: *yes*
 
 * *auto_pad*
 
   * **Description**: *auto_pad* how the padding is calculated. Possible values:
-    * *explicit*: use explicit padding values from `pads_begin` and `pads_end`.
-    * *same_upper (same_lower)* the input is padded to match the output size. In case of odd padding value an extra padding is added at the end (at the beginning).
+    * *explicit* - use explicit padding values from *pads_begin* and *pads_end*.
+    * *same_upper* - the input is padded to match the output size. In case of odd padding value an extra padding is added at the end.
+    * *same_lower* - the input is padded to match the output size. In case of odd padding value an extra padding is added at the beginning.
     * *valid* - do not use padding.
   * **Type**: string
-  * **Default value**: None
+  * **Default value**: explicit
   * **Required**: *no*
   * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
 
+
 * *group*
 
   * **Description**: *group* is the number of groups which *output* and *input* should be split into. For example, *group* equal to 1 means that all filters are applied to the whole input (usual convolution), *group* equal to 2 means that both *input* and *output* channels are separated into two groups and the *i-th output* group is connected to the *i-th input* group channel. *group* equal to a number of output feature maps implies depth-wise separable convolution.
-  * **Range of values**: integer value starting from 1
+  * **Range of values**: integer value starting from `1`
   * **Type**: int
-  * **Default value**: 1
+  * **Default value**: `1`
   * **Required**: *no*
 
 * *deformable_group*
 
   * **Description**: *deformable_group* is the number of groups which deformable values and *output* should be split into along the channel axis. Apply the deformable convolution using the i-th part of the offset part on the i-th out.
-  * **Range of values**: integer value starting from 1
+  * **Range of values**: integer value starting from `1`
   * **Type**: int
-  * **Default value**: 1
+  * **Default value**: `1`
   * **Required**: *no*
 
 **Inputs**:
 
-*   **1**: Input tensor of rank 3 or greater. Required.
+*   **1**: Input tensor of type *T* and rank 3, 4 or 5. Layout is NCZYX (number of batches, number of channels, spatial axes Z, Y, X). Required.
+
+*   **2**: Deformable values tensor of type *T* and rank 3, 4 or 5. Layout is *TBD*. Required.
```

Comment:
@lazarevevgeny @dmitry-gorokhov can you help me with understanding 'deformable values' input and what its layout should be?

Reply:
@iimironov maybe you can help here?

Reply:
MO should not modify the layout of weights in the DeformableConvolution (@iimironov , could you double check that?). So the layout should be as described https://github.com/apache/incubator-mxnet/blob/master/src/operator/deformable_convolution.cc#L64-L68

Reply:
Fixed.